### DISCUSSION TITLE
A new fixed-panel over the WF area

### DISCUSSION THREAD

**Discussion by GreyHound55** [Comfy-Org/ComfyUI_frontend#1113] [Ideas]
Updated ComfyUI yesterday, and a new "fixed" Panel (zoom, reset, etc...) appeared. Just like a few days ago, when the Queue fixed-panel appeared, a day later it was made "repositionable". Thank you!

Can this new Panel, please, also be made "repositionable" ?

I can't stress enough (like before), how the WF Area is at a premium and these Panels should not be stealing space from the WF Area. These Control Panels are secondary at best, in terms of priority. WF Area is where 99% of the focus/work occurs !

I'm still scratching my head after the last time ... Why the person who develops these, thinks this is a good idea ??? Almost like piecemeal development.

And please, don't take this wrong ... I love ComfyUI and greatly appreciate your team's efforts & how the development of Comfy is being progressed. Especially amazing, was how Flux was supported on Day 1 ! 👍

**Comment by LukeG89** [comment]
@GreyHound55 You can hide the new Graph Canvas menu in the Settings:

![show_graph_canvas_menu](https://github.com/user-attachments/assets/ef962a4f-0566-4a53-ad5e-2af62023018d)

**Reply by GreyHound55** [reply]
Thank you! 

But if you do want to see it ... can it be made repositionable? 

I have other panels active and have them reside in the UI's top-bar, so that they are readily useable, but also don't obscure the WF area 👍

**Reply by LukeG89** [reply]
I don't think that making it draggable would be convenient for this specific menu, maybe there could be an option to resize it and/or change its fixed position (top, middle and bottom).

Anyway, they recently implemented custom keybinding, so you can set your own shortcuts:

![keybinding_menu](https://github.com/user-attachments/assets/97bef322-b4cd-445b-87d3-600029e18cec)

There are all the Graph Canvas functionality you can set, so you can avoid having another panel on your screen and use your keyboard instead.

**Reply by GreyHound55** [reply]
Ok, that makes sense & thank you for the information 👍

This can be considered closed.
